Modules/_testcapimodule.c

```diff
@@ -4732,9 +4797,11 @@ static PyMethodDef TestMethods[] = {
     {"datetime_check_tzinfo",     datetime_check_tzinfo,         METH_VARARGS},
     {"make_timezones_capi",     make_timezones_capi,             METH_NOARGS},
     {"get_timezones_offset_zero",   get_timezones_offset_zero,   METH_NOARGS},
-    {"get_timezone_utc_capi",    get_timezone_utc_capi,            METH_VARARGS},
-    {"test_list_api",           test_list_api,                   METH_NOARGS},
-    {"test_dict_iteration",     test_dict_iteration,             METH_NOARGS},
+    {"get_timezone_utc_capi",    get_timezone_utc_capi,          METH_VARARGS},
+    {"get_date_fromtimestamp",   get_date_fromtimestamp,         METH_VARARGS},
+    {"get_datetime_fromtimestamp", get_datetime_fromtimestamp,   METH_VARARGS},
+    {"test_list_api",           (PyCFunction)test_list_api,      METH_NOARGS},
```

Comment:
I don't think an explicit cast to `PyCFunction` is needed. Both functions have compatible signatures.

Reply:
I didn't add this, it got pulled into the diff because of formatting changes I think. Either way, I'll remove it.

Reply:
Oh actually I think this was because this branch was originally created on Python 3.7 so I would know that the tests would have passed at the time. I think the function casts were dropped between then and now, but I pulled them in because of a merge conflict.

Fixed now.